common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerValidator.java

```diff
@@ -245,8 +216,7 @@ public String getCurrentActiveBrokerPackageName() {
      * @return PackageName of the broker
      */
     @NonNull
-    // TODO: Consolidate getCurrentActiveBrokerPackageName and getValidActiveBrokerPackageName into one.
-    public String getValidActiveBrokerPackageName() throws ClientException {
```

Comment:
Is there any other team that depends on this method, Intune?, Worth checking, once before removing.
Maybe mark is as deprecated with a comment that this would be removed in next version. and call the new method from inside it.

Reply:
this method is similar to getCurrentActiveBrokerPackageName, it was created with the intention to log more information.
it was only used in the controller of the new WPJ API.